runtime/Cpp/runtime/antlr4cpp-vs2022.vcxproj

```diff
@@ -1,652 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please don't remove the native project files. There are users of them (including myself).

I am not against modernizing the CMake build but this has nothing to with CMake.

Reply:
Sure. Also, Windows tests infrastructure use this file that's why build is failing.

Reply:
I can add these back in. I wonder about the intent here, though -- is CMake the primary build system? I would argue CMake should be the default, and that CI should be updated to use it. There are reasons not to keep native build files around, but I understand this would break projects which currently depend on them. Could we potentially add a deprecation of some sort?

Reply:
To be fair, CMake can generate project files for Visual Studio, but the CMake generated project files are not "portable" like the existing native ones because they use absolute paths. It would require quite some effort to switch to a CMake based build.